src/scene/graphics/shared/utils/buildContextBatches.ts

```diff
@@ -18,7 +19,7 @@ import type { GpuGraphicsContext } from '../GraphicsContextSystem';
 import type { GraphicsPath } from '../path/GraphicsPath';
 import type { ShapePath } from '../path/ShapePath';
 
-const buildMap: Record<string, ShapeBuildCommand> = {
+export const shapeBuilders: Record<string, ShapeBuildCommand> = {
```

Comment:
Could you register the shape extensions (extensions.add) instead of adding them here? Kinda defeats the purpose of making them extensions if you manually add them anyways.

Reply:
I've updated the PR to make this change!